.github/workflows/release.yml

```diff
@@ -27,8 +27,11 @@ jobs:
         uses: Swatinem/rust-cache@v2
         timeout-minutes: 5
         continue-on-error: true
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: "go.work"
       - name: Cargo Build
-        run: cargo build --release --bin forest --bin forest-cli --bin forest-tool
+        run: cargo build --release --bin forest --bin forest-cli --bin forest-tool --bin forest-wallet
```

Comment:
why not `make install`

Reply:
Good point. I'll go ahead and simplify it.

Reply:
@hanabi1224 It should be good now, I made it more generic and hopefully less likely to need additional maintenance in case we add more binaries.